### PR TITLE
Added links to posts & projects for next TWiR issue.

### PR DESCRIPTION
**Pull request by nasa42**
* Likely requires copy editing.
* Experimenting with new headings. A few of these can be merged, I think (though keeping them separate for now for feedback). Also, happy to revert to old headings.

### CODE REVIEW

**Comment by Gankro** [comment]
Oh wow, fantastic! @brson The Dream Is Real

File: drafts/2015-06-22-this-week-in-rust.md [hunk 1]
```diff
@@ -79,7 +101,9 @@ THING is a ...
 
 # Upcoming Events
 
-* [What?]
+* [6/23. Hannover](http://blog.thoughtram.io/rust/2015/06/17/anouncing-hanovers-second-rust-meetup.html)
```
Comment (Gankro) [inline drafts/2015-06-22-this-week-in-rust.md]: I'd like to see these expanded. A location isn't an event.

**Comment by Gankro** [comment]
The new headings seem fun; we're not a rigid formal document.

**Comment by brson** [comment]
This is fantastic! Thank you. You might also update the draft template with these sections.

**Pull request merged**